SU2_CFD/src/numerics_direct_mean.cpp

```diff
@@ -5120,72 +5095,60 @@ void CSourceAxisymmetric_Flow::ComputeResidual(su2double *val_residual, su2doubl
   su2double yinv, Pressure_i, Enthalpy_i, Velocity_i, sq_vel;
   unsigned short iDim, iVar, jVar;
   
-  bool implicit       = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
-  bool compressible   = (config->GetKind_Regime() == COMPRESSIBLE);
-  bool incompressible = (config->GetKind_Regime() == INCOMPRESSIBLE);
+  bool implicit = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
   
   if (Coord_i[1] > EPS) {
     
     yinv = 1.0/Coord_i[1];
     
-    if (compressible) {
-      sq_vel = 0.0;
-      for (iDim = 0; iDim < nDim; iDim++) {
-        Velocity_i = U_i[iDim+1] / U_i[0];
-        sq_vel += Velocity_i *Velocity_i;
-      }
+    sq_vel = 0.0;
+    for (iDim = 0; iDim < nDim; iDim++) {
```

Comment:
git is a little bit confused here. I assume that only the "if(compressible)"/if(incompressible)" cases have been removed here ?

Reply:
Yep, it's because I also re-indented that routine after modifying it so that it has the correct spacing for our style. Btw, you can now view the diffs in GitHub without the whitespace changes using the button controlling 'diff options' on the top right during the diff/review.